db/migrate/20230117072528_create_comment.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :AuthorId
+      t.integer :PostId
+      t.string :Text
```

Comment:
Great job @Mhamad-Raad 🤝

- [ ] [Optional] You may also make sure the Text is of type text. `t.text :text`.